### PR TITLE
Hotfix/RTENU-204 filter name tags

### PR DESCRIPTION
**Pull request by NinaDang97**
- Add new texts to landing page + search page when no result is found
- Fix tag's names

### CODE REVIEW

File: packages/frontend/src/assets/locales/fi/search.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
   "latest_first": "Uusin ensin",
   "last_searches": "VIIMEISET HAUT",
   "show_results": " Näytetään {{files}}/{{total}} tulosta",
+  "zero_result": "Hakusanalla \"{{query}}\" ei löytynyt tuloksia. Tarkista oikeinkirjoitus, kokeile toisella hakusanalla tai muokkaa valitsemiasi suodattimia. ",
```
Comment (mehtis) [inline packages/frontend/src/assets/locales/fi/search.json:17]: Is that space at the end on purpose?

File: packages/frontend/src/components/Tags/index.tsx [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
-  if (fromYear) {
-    yearRange += fromYear;
-    if (fromYear < toYear && toYear) {
-      yearRange += `- ${toYear}`;
+
+  const getYearTagName = (): string => {
+    const fromYear = formatYear(years[0]);
+    const toYear = formatYear(years[1]);
+    if (fromYear) {
+      return fromYear < toYear && toYear ? `${fromYear} - ${toYear}` : fromYear;
```
Comment (mehtis) [inline packages/frontend/src/components/Tags/index.tsx:17]: Shouldn't we first check that is there any toYear before comparing it to fromYear?

Comment (NinaDang97) [reply packages/frontend/src/components/Tags/index.tsx:17]: In frontend `toYear` calendar is disabled until `fromYear` is selected so it's reasonable to have it check bit afterwards

Comment (mehtis) [reply packages/frontend/src/components/Tags/index.tsx:17]: I meant this bit `fromYear < toYear && toYear`

Comment (mehtis) [reply packages/frontend/src/components/Tags/index.tsx:17]: Can `fromYear < toYear` be true if there is not `toYear`?

File: packages/frontend/src/pages/ContactInformation/InterchangeContactInformation/index.tsx [hunk 3]
```diff
@@ -1,12 +1,5 @@
...
 
 export const InterchangeContactInformation = () => {
-  const categoryNames = getSubCategoryData();
-
-  return (
-    <ProtectedContainerWrapper>
-      <PageTitleWrapper>{categoryNames.INTERCHANGE_CONTACT_INFORMATION}</PageTitleWrapper>
-    </ProtectedContainerWrapper>
-  );
+  return <ProtectedContainerWrapper></ProtectedContainerWrapper>;
```
[the remark this answers is not in the document]
Comment (NinaDang97) [reply packages/frontend/src/pages/ContactInformation/InterchangeContactInformation/index.tsx:4]: It's intentional for future use, some category pages already have separate content (e.g. Linjakaaviot has map component), some don't yet have but we never know :D

File: packages/frontend/src/components/Tags/index.tsx (hunk 2 above)
Comment (NinaDang97) [reply packages/frontend/src/components/Tags/index.tsx:17]: No, indeed `toYear` check is redundant

Comment (mehtis) [reply packages/frontend/src/components/Tags/index.tsx]: Do note that toYear is converted to 0 if it is null for that comparison. But presumably fromYear can't be less than 0, so this shouldn't be a problem.

**Review by noravux — APPROVED** [review]

**Pull request merged**